pkg/aspect/config/quotas.proto

```diff
@@ -32,7 +33,15 @@ message QuotasParams {
         // Map of quota_descriptor label name to attribute expression. At run time each
         // expression will be evaluated to determine the value provided to the aspect. The
         // result of evaluating the expression must match the ValueType of the label in the quota_descriptor.
-        map<string, string> labels = 3;
+        map<string, string> labels = 2;
+
+        // The upper limit for this quota.
+        int64 max_amount = 3;
```

Comment:
This is the number of quota configs quota manager should get.

1. most specific subject within every scope
2. quota combine step which will be given all these quotas to combine.
3. dispatch those quotas 

What visibility are you looking for in inside the quota manager ?